src/iris_grib/_save_rules.py

```diff
@@ -230,12 +230,14 @@ def dx_dy(x_coord, y_coord, grib):
 
 
 def scanning_mode_flags(x_coord, y_coord, grib):
-    eccodes.codes_set_long(
-        grib, "iScansPositively", int(x_coord.points[1] - x_coord.points[0] > 0)
-    )
-    eccodes.codes_set_long(
-        grib, "jScansPositively", int(y_coord.points[1] - y_coord.points[0] > 0)
-    )
+    x_positive = x_coord.points[1] - x_coord.points[0] > 0
+    y_positive = y_coord.points[1] - y_coord.points[0] > 0
+    scanningMode = 0
+    if not x_positive:
+        scanningMode |= 0x80  # "bit 1" has negative sense : set=decreasing
+    if y_positive:
+        scanningMode |= 0x40  # "bit2" has positive sense : set=increasing
+    eccodes.codes_set_long(grib, "scanningMode", scanningMode)
```

Comment:
There are still some references to `...ScansNegatively` and `...ScansPositively` in the Iris-grib codebase; apparently limited to GRIB-1 related code. Is this a problem?

Reply:
No, I think it's OK to _read_ them, but they can no longer be written.
And probably nest to keep GRIB1 code as it was, as with the old "indicatorOfUnitOfTimeRange" keyname which is still the version needed for GRIB1.